index.html

```diff
@@ -305,15 +309,9 @@ <h2 id="resultsQuestion">Question</h2>
             }
 
             if (update.serial === update.max_serial) {
-                if (!isConfigured()) {
-                    refresh("configurePage");
-                } else if (!hasSelfVoted()) {
-                    refresh("votePage");
-                } else {
-                    refresh("resultsPage");
-                }
+                updateScreen()
             }
-        })
+        }).then(updateScreen)
```

Comment:
that would call updateScreen() two times in case there are some updates, right?

no problem, just asking for understanding :)

Reply:
yes it would, but makes no big difference here